nautobot_golden_config/tables.py

```diff
@@ -22,10 +22,10 @@
 
 ALL_ACTIONS = """
 {% if backup == True %}
-    {% if record.configcompliance_set.first.rule.config_type == 'json' %}
+    {% if record.config_type == 'json' %}
```

Comment:
These are working because `record` is changing to the `device` object is that correct?